src/Makefile

```diff
@@ -492,6 +492,22 @@ mlperf_training:
 	cd $(BINDIR)/$(BINSUBDIR) && sh ./install_mlperf_and_virtual_environments_training.sh | tee $(BINDIR)/$(BINSUBDIR)/mlperf_install_training.log
 	cd $(BINDIR)/$(BINSUBDIR) && rm install_mlperf_and_virtual_environments_training.sh
 
+cuda_samples:
```

Comment:
This is an interesting little bash loop, did you get AI to write this for you?

Reply:
If so -you need a better AI - I don't think we need this.
Here is what AI gave me as an alternative. I have done stuff like this before with the wildcard keyword.

<img width="725" alt="image" src="https://github.com/user-attachments/assets/9d42da9d-5265-4b2c-a490-b410c8959d2d" />